doc/source/whatsnew/v0.18.2.txt

```diff
@@ -528,3 +528,5 @@ Bug Fixes
 
 - Bug in ``Categorical.remove_unused_categories()`` changes ``.codes`` dtype to platform int (:issue:`13261`)
 - Bug in ``groupby`` with ``as_index=False`` returns all NaN's when grouping on multiple columns including a categorical one (:issue:`13204`)
+
```

Comment:
note the reason we leave space in the bug fix section is to avoid conflicts (IOW put up higher in a blank space). @jorisvandenbossche you prob want to move on merge.